src/socket/server.cr

```diff
@@ -2,12 +2,33 @@ class Socket
   module Server
     # Accepts an incoming connection and returns the client socket.
     #
+    # ```
+    # require "socket"
+    #
+    # server = TCPServer.new(2202)
+    # while true
```

Comment:
`loop do`?

Reply:
Both are fine. I'm fact, I'm starting to think we should remove `loop`, it adds absolutely nothing and it's just another way of doing something.

Reply:
well I guess it was useful when it had its block argument :P